docs/meshstack.azure.landing-zones.md

```diff
@@ -57,12 +57,28 @@ Blueprints are versioned in Azure and can be managed via the Azure Portal. To av
 - Existing projects with this Landing Zone will get their Blueprint updated to this version on the next [replication](./meshcloud.tenant.md)
 - Newly created projects will get the latest Blueprint version assigned (possibly higher then the version configured here)
 
+#### Use User Assigned Managed Identity (UAMI)
+
+This flag allows you to use a User Assigned Managed Identity instead of the standard System Assigned Managed identity during the assignment of your blueprint. For more details on their differences, refer to the [Azure Documentation](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview#how-does-the-managed-identities-for-azure-resources-work)).
```

Comment:
```suggestion
This flag allows you to use a User Assigned Managed Identity (UAMI) instead of the standard System Assigned Managed Identity (SAMI) during the assignment of your blueprint. For more details on their differences, refer to the [Azure Documentation](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview#how-does-the-managed-identities-for-azure-resources-work)).
```